clients/fides-js/src/components/index.ts

```diff
@@ -1,2 +1 @@
-export { default as NoticeToggles } from "./NoticeToggles";
```

Comment:
this was from my experiments of trying to use the toggles in privacy center, and isn't used